sensor/init-tls-certs/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	destinationDir  = "/tmp/certs"
```

Comment:
How about injecting these values via environment variables so that we have these paths specified in a single file (in the manifests for the workloads).